docs/src/.vitepress/config.mts

```diff
@@ -39,10 +40,12 @@ export default defineConfig({
       { text: 'Getting Started', link: '/getting_started' },
       { text: 'Markdown', link: '/markdown-examples' },
       { text: 'Code', link: '/code_example' },
-      { text: 'API', link: '/api' }
+      { text: 'API', link: '/api' },
+      { text: 'versions',
```

Comment:
```suggestion
      { text: 'REPLACE_ME_DOCUMENTER_VITEPRESS',
```
and then we add the current version (be it dev, preview, or v*) to it?

Reply:
mmm... note that the `items` part is the thing that needs to be replaced.

Reply:
I thought you had some code in the component to make items dynamic?